nixos/tests/postgis.nix

```diff
@@ -9,15 +9,15 @@ import ./make-test.nix ({ pkgs, ...} : {
       { pkgs, config, ... }:
 
       {
-        services.postgresql = let mypg = pkgs.postgresql95; in {
+        services.postgresql = let mypg = pkgs.postgresql100; in {
             enable = true;
             package = mypg;
-            extraPlugins = [ (pkgs.postgis.override { postgresql = mypg; }).v_2_2_1 ];
-            initialScript =  pkgs.writeText "postgresql-init.sql"
-          ''
-          CREATE ROLE postgres WITH superuser login createdb;
-          '';
-          };
+            extraPlugins = [ (pkgs.postgis.override { postgresql = mypg; }).v_2_4_0 ];
+            #initialScript =  pkgs.writeText "postgresql-init.sql"
+            #''
+            #CREATE ROLE postgres WITH superuser login createdb;
+            #'';
```

Comment:
I think those lines can be completly removed singe they are not necessary anymore.